### ISSUE TITLE
I need your help to translate some messages to your local language.

### ISSUE THREAD

**Issue by zsnmwy** [zsnmwy/ONI-geyser-calculator#14]
@Kroning @Neoazureus

Thanks for your translation.

Add now I add some new messages on the web site.  https://onical.ga

![](https://i.imgur.com/87VtHwl.jpg)

These messages can tell us what can they do in detail.

I think it's useful.

Now I need your help to translate these messages to your local language. 

```
        use_up_information:'Machines & Plants Ratio',
        Bristle_Blossom:'Bristle Blossom Comsumption ${0} g/s, Number ${1}',
        Sleet_Wheat:'Sleet Wheat Comsumption ${0} g/s, Number ${1}',
        Pincha_Pepper:'Pincha Pepper Comsumption ${0} g/s, Number ${1}',
        Thimble_Reed:'Thimble Reed Comsumption ${0} g/s, Number ${1}',
        Oil_Well:'Oil Well Comsumption ${0} g/s, Number ${1}',
        Electrolyzer:'Electrolyzer Comsumption ${0} g/s, Number ${1}',
        Fertilizer_Synthesizer:'Fertilizer Synthesizer Comsumption ${0} g/s, Number ${1}',
        Carbon_Skimmer:'Carbon Skimmer Comsumption ${0} g/s, Number ${1}',

```

Thanks.  :）

**Comment by Kroning** [comment]
Yeah, it's useful! I will add russian translation a little bit later (1-3 days I suppose).

**Comment by henrimitte** [comment]
Hello. I translated the page to Brazilian Portuguese (PT-BR). The icon that should be used is the [Brazilian Flag](https://twemoji.maxcdn.com/2/72x72/1f1e7-1f1f7.png). The main countries that speak Portuguese are Brazil and Portugal. The file with the translated content are attached.

[pt-br-template.txt](https://github.com/zsnmwy/ONI-geyser-calculator/files/2848844/pt-br-template.txt)
![brasil](https://user-images.githubusercontent.com/40926031/52533661-313f1f80-2d1e-11e9-9886-3bd2b62812d3.png)
